### PR TITLE
Added support for IMU aided GPS systems like the Applanix POS/MV, 

### PR DESCRIPTION
**Pull request by valschmidt**
...whose NMEA strings typically begin '$IN'. (e.g. $INGGA).

### CODE REVIEW

**Comment by valschmidt** [comment]
In this commit, I also added support for VTG messages, which give CMG and SOG. This is sometimes more common than RMC messages (although RMC messages have the benefit of a date/time stamp, which VTG messages do not).

**Comment by evenator** [comment]
I don't have access to hardware I can test this with. @valschmidt can you make a small bag file to test this against?

File: src/libnmea_navsat_driver/parser.py [hunk 1]
```diff
@@ -120,13 +120,17 @@ def convert_deg_to_rads(degs):
         ("longitude_direction", str, 6),
         ("speed", convert_knots_to_mps, 7),
         ("true_course", convert_deg_to_rads, 8),
+        ],
+    "VTG":[
+        ("true_course",safe_float,1),
+        ("speed",convert_knots_to_mps,5)
```
Comment (evenator) [inline src/libnmea_navsat_driver/parser.py]: [nit] Please put spaces after the commas, i.e.

```python
        ("true_course", safe_float, 1),
        ("speed", convert_knots_to_mps, 5)
```

File: src/libnmea_navsat_driver/driver.py [hunk 2]
```diff
@@ -49,6 +49,8 @@ def __init__(self):
 
         self.time_ref_source = rospy.get_param('~time_ref_source', None)
         self.use_RMC = rospy.get_param('~useRMC', False)
+        self.valid_fix = False
+        self.last_valid_fix_time = None
```
Comment (evenator) [inline src/libnmea_navsat_driver/driver.py]: I see you set this variable, but I never see you use it. Is it necessary?

Comment (valschmidt) [reply src/libnmea_navsat_driver/driver.py]: No, not for this commit. This was an incomplete effort to fix a bug in the parser code. The parser code captures the system time to get the date for the GPS fix. But there is a real chance that the string will be parsed within the error between the system time and the GPS fix at the turn of the UTC day, which will put the time stamp 24 hours off. The fix doesn't belong here, but rather in parser.py.

File: src/libnmea_navsat_driver/parser.py (hunk 1 above)
Comment (valschmidt) [reply src/libnmea_navsat_driver/parser.py]: Done.

File: src/libnmea_navsat_driver/driver.py (hunk 2 above)
Comment (valschmidt) [reply src/libnmea_navsat_driver/driver.py]: Variable removed in latest commit.

**Comment by valschmidt** [comment]
I have put a bag of Sentence type messages containing strings generated from a POS/MV at the following link:
https://unh.box.com/s/wgzg3kmpy1xweopi6uk7ffku4d6w6l08
There are INGGA, INHDT, INVTG, INGST, INZDA and a few other proprietary string types. I'm sorry, I don't have one with INRMC and my gear is in the field at the moment.

**Comment by evenator** [comment]
@valschmidt I think you need to change the sharing settings on that file. I can't access it.

**Comment by valschmidt** [comment]
I apologize. Please try again. I think (hope) it's fixed.

**Comment by evenator** [comment]
I finally got a chance to test this. It looks great. Thanks for contributing!

**Pull request merged**